azure_billing/scrape.py

```diff
@@ -74,7 +74,7 @@ def data(enrollment, token, month=None):
 
     df = convert_json_df(raw)
 
-    c = Counter('azure_costs', "Costs billed to Azure Enterprise Agreement {}".format(enrollment))
+    c = Counter(metric_name, "Costs billed to Azure Enterprise Agreement {}".format(enrollment))
```

Comment:
Also, the description could feature a unit (or one could think of appending a unit such as `euros` or `dollars` to the metric name to be consistent with the metric naming convention of having a unit at the end.